src/json-patch-ot-agent.js

```diff
@@ -72,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
```

Comment:
I don't understand. You need to reset pending and ackLocalVersion to prevent unnecessary retransmission.